src/scheduler/data_types.h

```diff
@@ -573,6 +573,7 @@ struct job_info
 	unsigned topjob_ineligible:1;	/* Job is ineligible to be a top job */
 
 	char *job_name;			/* job name attribute (qsub -N) */
+	int server_sock;		/* server socket associated with this job */
```

Comment:
can we add it to resource_resv instead? one less indirection to go through when accessing it, and we might need it for reservations as well.

Reply:
I thought of this originally and did not add it for reservations also because they are shared across all the servers and we learn more when we implement them. Due to this I thought it is better add this to job_info object itself.

Reply:
Well we haven't designed reservations yet, so I think we shouldn't assume that they will be shared, but at least while confirming a reservation, the sched does need to know which server sent it the reservation submit request so that it can respond it to that server, right?